### ISSUE TITLE
Remove Previous and Next links

### ISSUE THREAD

**Issue by pcav** [qgis/OLD-QGIS-Website-see-QGIS-Website-for-current-one#6]
Previous and Next links should be removed from top menu (useful for manuals, not for a website)

**Comment by pcav** [comment]
Removing this from layout.html does it:

```
        {% block sidebarrel %}
          {% include "relations.html" %}
        {% endblock %}
```

**Comment by pcav** [comment]
Fixed in sphinx-bootstrap-theme v 0.2.8
